src/local-history/local-history-manager.ts

```diff
@@ -78,43 +76,43 @@ export class LocalHistoryManager {
     /**
      * View all history of the active editor.
      */
-    public viewAllForActiveEditor(textEditor: vscode.TextEditor): void {
+    public viewAllHistoryForActiveEditor(textEditor: vscode.TextEditor): void {
```

Comment:
I don't really like the naming of this method, or the command itself.
I think we can be more concise with naming variables, ex:

```ts
/**
* View the local-history for the given editor.
*/
viewHistory(editor: vscode.TextEditor): void { }
```